packages/react-components/src/link/src/TextLink.jsx

```diff
@@ -107,7 +107,7 @@ export function InnerTextLink(props) {
         },
         text: {
             size,
-            className: "o-ui-link-text"
+            className: "o-ui-link-text o-ui-text-color-inherit"
```

Comment:
Hmmm? Since you always render `o-ui-text-color-inherit` why don't you set it in the CSS then?